### PR TITLE
Python: various new packages

### PR DESCRIPTION
**Pull request by NikolaMandic**
###### Things done
- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [ ] NixOS
  - [ ] OS X
  - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -104,6 +104,166 @@ in modules // {
...
+    propagatedBuildInputs = with self; [ ];
+
+    src = pkgs.fetchurl {
+      url = "mirror://pypi/P/PreTrainingChain/${name}.zip";
+      sha256 = "be39272c4b5bcefcf97c1d6000c5e11a48b85053efe15fc9a2dbd0a1621b64c3";
+    };
+  };
+
+  zChainer = buildPythonPackage rec {
+    name = "PreTrainingChain-${version}";
```
Comment (groxxda) [inline pkgs/top-level/python-packages.nix]: :boom:

Comment (NikolaMandic) [inline pkgs/top-level/python-packages.nix]: I'll ping when stuff gets better

File: pkgs/top-level/python-packages.nix [hunk 2]
```diff
@@ -104,6 +104,185 @@ in modules // {
...
+
+  scikit-cuda = buildPythonPackage rec {
+    name = "scikit-cuda-${version}";
+    version = "0.5.1";
+    disabled = isPy3k;
+
+    meta = {
+      description = "Python interfaces to many of the functions in the CUDA device/runtime";
+      homepage    = "https://github.com/lebedov/scikit-cuda/";
+      license     = licenses.mit; #has no licence
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: https://github.com/lebedov/scikit-cuda/blob/master/docs/source/license.rst

**Comment by FRidh** [comment]
Please follow https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md
As soon as that's done, you can cc me.

**Comment by FRidh** [comment]
@NikolaMandic it would be nice getting this in. Could you split the commit? I'm closing this for now. Please reopen if it looks ready to be merged.

**Pull request closed without merging**